Dispatcher/Controllers/Homepage/HomepageViewModel.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  HomepageViewModel.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 27/04/2023.
+//
+
+import Foundation
+import UIKit
```

Comment:
In any file that we don't actually use UI elements - we can spare importing the UIKit.
Importing unnecessary modules can increase the build time and compile time, as the Swift compiler needs to load and process more code